api/python/tests/test_cli.py

```diff
@@ -41,5 +42,38 @@ def test_push(self):
                 main.main(('push', '--dir', tmp_dir, name))
 
                 mocked_package_class.assert_called_once_with(quilt3.Package)
-                mocked_set_dir.assert_called_once_with('.', tmp_dir)
+                mocked_set_dir.assert_called_once_with('.', tmp_dir, meta=None)
```

Comment:
instead of changing this call we should have a default falsy value for `meta={}` for example; it might even get rid of the `if meta` special case in main.py.

Reply:
we can't set a default for `meta` in main.py because 
```
meta = json.loads(meta)
```
expects a `str` not a `dict` nor a `NoneType`.  Also, it will still require changing the expected call param you pointed at above.